content/docs/command-reference/remote/modify.md

```diff
@@ -857,69 +857,70 @@ Read more about by expanding the WebHDFS section in
 by HDFS. Read more about by expanding the WebHDFS section in
 [`dvc remote add`](/doc/command-reference/remote/add#supported-storage-types).
 
-- `url` - remote location:
+- `url` - remote location.
 
   ```dvc
   $ dvc remote modify myremote url webhdfs://user@example.com/path
   ```
 
-- `user` - user name to access the remote, can be empty in case of using `token`
-  or if using a `HdfsCLI` cfg file. May only be used when Hadoop security is
-  off. Defaults to current user as determined by `whoami`.
+  Only provide the `user` parameter if you are not using `kerberos` or `token`
+  authentication, since those authentication methods already contain the user
+  information.
```

Comment:
A bit confused. So there's no more `user` parameter? This removes that bullet.

I see it's removed from the schema in https://github.com/iterative/dvc/pull/6936/files but it's still used on the test. 

@gudmundur-heimisson @efiop thanks

Reply:
What about the `HdfsCLI` cfg file? Is that no longer a viable alternative?

Reply:
p.s. for now I'm reinstating it in https://github.com/iterative/dvc.org/pull/3019.

Reply:
In the test it is read out of the URL string. It feels confusing to me to be able to provide the user in both the URL string and also as an explicit parameter so I just removed it as a parameter, but I don't feel that strongly about it.

As a side-note for WebHDFS if the cluster is secured you will never be using this parameter anyway because you must use kerberos or the token (unless you have a Cloudera cluster which can allow LDAP, but that is not part of the Apache standard).

If my understanding is correct, because of this line if we allow it as an explicit parameter as well it would override whatever is in the URL string, which we should probably document somewhere:

https://github.com/iterative/dvc/blob/e0fd0790fc77b19694aabef3c94b52e7cd0135f1/dvc/fs/__init__.py#L120

Reply:
Thanks. Still confused though, so is the `user` param deprecated currently? @efiop